cibuildwheel/platforms/windows.py

```diff
@@ -329,7 +329,7 @@ def setup_python(
                 "pip",
                 "install",
                 "--upgrade",
-                "build[virtualenv]",
+                "build[virtualenv, uv]",
```

Comment:
This will install another copy of there is already an external one. I think we try to avoid that everywhere else.

Reply:
#2691 opened as an alternative